src/components/Auth/ChangePassword/ChangePassword.react.js

```diff
@@ -304,6 +292,14 @@ class ChangePassword extends Component {
 
     const PasswordClass = [`is-strength-${newPasswordScore}`];
 
+    const labelStyle = {
```

Comment:
Move this out of the component into the style object. Everything except the ```color: themeForegroundColor``` part